cherab/generomak/machine/data/equilibrium/generomak_equilibrium.json

```diff
@@ -0,0 +1,4630 @@
+        -1.25,
+        -1.2,
+        -1.15,
+        -1.1,
+        -1.05,
+        -1.0,
+        -0.95,
+        -0.9,
+        -0.85,
+        -0.8,
+        -0.75,
+        -0.7,
+        -0.65,
+        -0.6,
+        -0.55,
+        -0.5,
+        -0.45,
+        -0.4,
+        -0.35,
+        -0.3,
+        -0.25,
+        -0.2,
+        -0.15,
+        -0.1,
+        -0.05,
+        -1.38778e-17,
+        0.05,
+        0.1,
+        0.15,
+        0.2,
+        0.25,
+        0.3,
+        0.35,
+        0.4,
+        0.45,
+        0.5,
+        0.55,
+        0.6,
+        0.65,
+        0.7,
+        0.75,
+        0.8,
+        0.85,
+        0.9,
+        0.95,
+        1.0,
+        1.05,
+        1.1,
+        1.15,
+        1.2,
+        1.25,
+        1.3,
+        1.35,
+        1.4,
+        1.45,
+        1.5,
+        1.55
+    ],
+    "limiter_polygon": [
+        [
+            1.845,
+            1.727102295,
+            0.990566772,
+            0.990566772,
+            0.755,
+            0.755,
+            0.975565369,
+            1.6828032229999998,
+            1.964,
+            2.023902466,
+            2.069891357,
+            2.1206196289999997,
+            2.161329834,
+            2.192982422,
+            2.228721191,
+            2.259991211,
+            2.2868015140000004,
+            2.3082133789999997,
+            2.322440674,
+            2.3386960450000003,
+            2.3500270999999997,
+            2.355582031,
+            2.36036084,
+            2.3610952149999997,
+            2.35839502,
+            2.35072876,
+            2.342213623,
+            2.3266232909999998,
+            2.305357422,
+            2.279498535,
+            2.2590996089999997,
+            2.228743896,
+            2.206925781,
+            2.172088135,
+            2.13348291,
+            2.102367432,
+            2.060595947
+        ],
+        [
+            -1.31,
+            -1.75,
+            -1.75,
+            -1.43,
+            -0.705,
+            1.175,
+            1.49,
+            1.49,
+            0.834882751,
+            0.781796082,
+            0.736593079,
+            0.6809180300000001,
+            0.630384766,
+            0.586222839,
+            0.529278137,
+            0.470608307,
+            0.410247284,
+            0.351394562,
+            0.303779938,
+            0.234423233,
+            0.16623642,
+            0.117408417,
+            0.041775253,
+            -0.02012863,
+            -0.096360207,
+            -0.183961243,
+            -0.24792569,
+            -0.33580651899999997,
+            -0.428234711,
+            -0.518649048,
+            -0.579311096,
+            -0.657460205,
+            -0.706716553,
+            -0.776043274,
+            -0.8422613529999999,
+            -0.889208679,
+            -0.9450545650000001
+        ]
+    ],
+    "b_vacuum_radius": 1.48,
+    "b_vacuum_magnitude": 2,
+    "time": 0.0,
+    "f_profile": [
```

Comment:
This profile is inconsistent with the B<sub>vac0</sub> value. First of all, it's negative, while B<sub>vac0</sub> is positive. Also, at ψ<sub>norm</sub> = 1, it should give B<sub>vac0</sub> * R<sub>0</sub> = 2.96, but gives -2.93 instead. Can you please renormalize it?

Reply:
Thanks for spotting this. I corrected the coordinate system and recalculated f and q profiles.

Reply:
Thanks, @Mateasek. Everything seems fine now.